File: stress/t/flt_trig.c
#include <stdio.h>
#include <math.h>
int main(void){
  double x,s,c,t;
  x=0.7; s=sin(x); c=cos(x); t=tan(x);
  printf("s= %f\n",s);
  printf("c= %f\n",c);
  printf("t= %f\n",t);
  x=3.14159; s=sin(x); c=cos(x);
  printf("s= %f\n",s);
  printf("c= %f\n",c);
  return 0;
}

File: stress/t/flt_trig.f
      PROGRAM FTRIG
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      X=0.7D0
      S=SIN(X)
      C=COS(X)
      T=TAN(X)
      WRITE(*,'(A,F0.6)') 's= ',S
      WRITE(*,'(A,F0.6)') 'c= ',C
      WRITE(*,'(A,F0.6)') 't= ',T
      X=3.14159D0
      S=SIN(X)
      C=COS(X)
      WRITE(*,'(A,F0.6)') 's= ',S
      WRITE(*,'(A,F0.6)') 'c= ',C
      END
